frontend/src/routes/(authenticated)/project/create/+page.svelte

```diff
@@ -112,9 +111,8 @@
     true,
     true,
   );
-  run(() => {
-    $asyncCodeError = $codeIsAvailable ? undefined : $t('project.create.code_exists');
-  });
+  const asyncCodeError = writable<string | undefined>($t('project.create.code_exists'));
+  onMount(() => codeIsAvailable.subscribe((_) => ($asyncCodeError = undefined)));
```

Comment:
This should only be clearing the error if `$codeIsAvailable` is `true`.

I think this should work:
```suggestion
  const asyncCodeError = $codeIsAvailable ? undefined : $t('project.create.code_exists');
```

Reply:
asyncCodeError needs to be a store in order for the `codeErrors` derived store in the next line to work, so I took your suggestion and adapted it. Done in commit 145f6182.

Reply:
Noticed a bug here, but it turns out it was already in production.
Fixed it anyway 😆